database/skylink_test.go

```diff
@@ -0,0 +1,45 @@
+package database
+
+import "testing"
+
+// TestValidateSkylink ensures validateSkylink properly returns the skylink hash
+func TestValidateSkylink(t *testing.T) {
+	tests := []struct {
+		name  string
+		in    string
+		out   string
+		valid bool
+	}{
+		{
+			name:  "valid0",
```

Comment:
nit: Think we can remove the name field and just use the `in` value in the error message.

Reply:
Good idea, we will have a clue what's going on before even opening the code.